documentation/src/main/asciidoc/topics/ref_rest_xsite_cache_managers.adoc

```diff
@@ -15,39 +15,42 @@ GET /rest/v2/cache-managers/{cacheManagerName}/x-site/backups/
 include::json_examples/xsite_backup_status.json[]
 ----
 
-The `status` field has the following values:
+.Returned Status
 
-* `online`: all caches are online in the backup site.
-* `offline`: all caches are offline in the backup site.
-* `mixed`: some caches are online and others offline, and their names will be listed in the `online` and `offline` arrays respectively.
+|===
+|Value |Description
 
+|`online`
+| All caches in the backup location are online.
```

Comment:
same as above. this is how the local caches see the remote site. 
CACHE_1 sees NYC-2 online (i.e. is able to send the updates) but CACHE_2 isn't.